docs/modules/user-guide/pages/api/examples.adoc

```diff
@@ -58,17 +58,20 @@ $ curl -v -X POST {sbomer-url}/api/{sbomer-latest-api-version}/generations \
 
 TIP: See xref:generation/configuration/pnc-analysis.adoc[PNC analysis configuration].
 
-Request manifest generation for following zips:
+Request manifest generation for following zips, storing the results under PNC miletone ID 0:
 
 - https://download.host.com/staging/product-a/release-b/first.zip
 - https://download.host.com/staging/product-a/release-b/second.zip
 
+Please note that you must provide a valid PNC miletone ID or analysis will fail.
```

Comment:
Please note that you must provide a valid PNC milestone id or the analysis will fail.